app/Actions/Album/Move.php

```diff
@@ -12,16 +12,16 @@ class Move extends Action
 	/**
 	 * Moves the given albums into the target.
 	 *
-	 * @param Album|null $targetAlbum
-	 * @param Collection $albums
+	 * @param Album|null        $targetAlbum
+	 * @param Collection<Album> $albums
 	 *
 	 * @throws ModelNotFoundException
 	 * @throws ModelDBException
 	 */
 	public function do(?Album $targetAlbum, Collection $albums): void
 	{
 		// Move source albums into target
-		if ($targetAlbum) {
+		if ($targetAlbum != null) {
```

Comment:
```suggestion
		if ($targetAlbum !== null) {
```

I am confused that PhpStan forbids the short syntax `if ($targetAlbum)`  which should be safe here due to type hinting, but allows the weak comparison `!= null`.

Reply:
Yes, but PhpStan enforces booleans in if-statements and ternary operators, thus the short syntax is not allowed.

Reply:
Agreed. But why are you using the weak comparison?